pkg/engine/renderer/functions.go

```diff
@@ -0,0 +1,9 @@
+package renderer
+
+import "gopkg.in/yaml.v2"
```

Comment:
most of our code uses "sigs.k8s.io/yaml" (except test harness)
Is there a reason to pick this package?

Reply:
I think this package does "straight to YAML" marshalling, while `sigs.k8s.io/yaml` is "object -> json -> yaml". I believe the reason for the latter is to support `json:` [struct tags](https://godoc.org/encoding/json) but we do not need this in this case.